index.js

```diff
@@ -0,0 +1,70 @@
+// Create Dom element
+
+const addBooks = localStorage.getItem('adbooks');
+let books = [];
+
+if (addBooks) {
+  books = JSON.parse(addBooks);
+} else {
+  const bookData = [
+    {
+      id: 1,
+      name: 'Lorem ipsum',
+      author: 'Testeroo Testyy',
+    },
+  ];
```

Comment:
Please ensure that the hard-coded book is removed from your code. As you are to generate the data dynamically.

Reply:
Done